draft-ietf-tls-tls13.md

```diff
@@ -3541,20 +3551,20 @@ C, and D.
 
 #  IANA Considerations
 
-[[TODO: Update https://github.com/tlswg/tls13-spec/issues/62]]
 [[TODO: Rename "RSA" in TLS SignatureAlgorithm Registry
 to RSASSA-PKCS1-v1_5 ]]
 
 This document uses several registries that were originally created in
 {{RFC4346}}. IANA has updated these to reference this document. The registries
-and their allocation policies (unchanged from {{RFC4346}}) are listed below.
+and their allocation policies are below:
 
--  TLS Cipher Suite Registry: Future values with the first byte in
-  the range 0-191 (decimal) inclusive are assigned via Standards
-  Action {{RFC2434}}.  Values with the first byte in the range 192-254
-  (decimal) are assigned via Specification Required {{RFC2434}}.
+- TLS Cipher Suite Registry: Values with the first byte in the range
+  0-254 (decimal) are assigned via Specification Required {{RFC2434}}.
   Values with the first byte 255 (decimal) are reserved for Private
-  Use {{RFC2434}}.
+  Use {{RFC2434}}. IANA [SHALL add/has added] a "Recommended" column
```

Comment:
I thought we had agreed that this would apply to 1.2 as well. But happy to do it either way.